File: setima/c/2.c
#include <stdio.h>

int main() {
    int matriz[4][5];

    for (int i = 0; i < 4; i++) {
        for (int j = 0; j < 5; j++) {
            printf("Digite o elemento da posição (%d, %d): ", i + 1, j + 1);
            scanf("%d", &matriz[i][j]);
        }
    }

    // Apresentar a matriz
    for (int i = 0; i < 4; i++) {
        for (int j = 0; j < 5; j++) {
            printf("%d ", matriz[i][j]);
        }
        printf("\n");
    }

    return 0;
}

File: setima/python/2.py
matriz = [[0] * 5 for _ in range(4)]

for i in range(4):
    for j in range(5):
        elemento = int(input(f"Digite o elemento da posição ({i + 1}, {j + 1}): "))
        matriz[i][j] = elemento

# Apresentar a matriz
for linha in matriz:
    print(linha)
